docs/whatsnew/2.1.rst

```diff
@@ -0,0 +1,215 @@
+By transforming this coordinate to `~sunpy.coordinates.frames.HeliocentricInertial`, we can see that Mars's actual velocity is larger::
+
+    >>> mars.heliocentricinertial
+    <SkyCoord (HeliocentricInertial: obstime=2021-01-01T00:00:00.000): (lon, lat, distance) in (deg, deg, AU)
+        (-9.91128592, 1.77496469, 1.50936573)
+     (d_lon, d_lat, d_distance) in (arcsec / s, arcsec / s, km / s)
+        (0.04174239, 0.00060976, 19.54950058)>
+    >>> mars.heliocentricinertial.velocity.norm()
+    <Quantity 49.68592218 km / s>
+
+See :ref:`sunpy-coordinates-velocities` for more information.
+
+.. _whatsnew-2.1-reprojecting:
+
+Alternatives for reprojecting a Helioprojective map
+===================================================
+The typical observation in `~sunpy.coordinates.frames.Helioprojective` coordinates does not contain full 3D information for the sources of emission, so an assumption needs to be made when transforming such coordinates to other coordinate frames.
+By default, SunPy assumes that the emission is coming from the surface of the Sun, which enables reprojections such as in the example :ref:`sphx_glr_generated_gallery_map_transformations_reprojection_different_observers.py`.
+However, this assumption is not appropriate for some observations, e.g., from coronagraphs.
+
+There is now a context manager (:meth:`~sunpy.coordinates.frames.Helioprojective.assume_spherical_screen`) to override the default assumption such that any 2D coordinates are interpreted as being on the inside of a large spherical screen.
+See the following example for how this context manager enables alternative reprojections.
+
+.. minigallery:: sunpy.coordinates.Helioprojective.assume_spherical_screen
+
+.. _whatsnew-2.1-performance:
+
+Performance improvements
+========================
+Several functions in `sunpy.map.maputils` have been significantly sped up with improved algorithms.
+
+In addition, `sunpy.map.Map.wcs` is now cached when the map metadata remains unchanged, significantly improving performance in applications which make mutiple requests for the map WCS (e.g. plotting), and reducing the number of repeated warnings thrown when metadata is missing.
+
+.. _whatsnew-2.1-goes1617:
+
+Support for GOES-16 and GOES-17 X-Ray Sensor (XRS) data
+=======================================================
+We have now included support for the GOES-16 and GOES-17 XRS 1-second data. This data can now be queried and downloaded with `sunpy.net.Fido` and the files read in and analysed as a `sunpy.timeseries.TimeSeries`.
+
+See the following example for how this can be achieved :ref:`sphx_glr_generated_gallery_acquiring_data_goes_xrs_example.py`
+
+.. _whatsnew-2.1-python:
+
+Increase in required package versions
+=====================================
+We have bumped the minimum version of several packages we depend on; these are the new minimum versions for sunpy 2.1:
+
+- python 3.7
+- astropy 4.0
+- scipy 1.2
+- parfive 1.1
+- drms 0.6.1
+- matplotlib 3.1.1
```

Comment:
Wait...  I thought we were bumping matplotlib with SunPy 3.0, not for SunPy 2.1...

Reply:
```suggestion
- matplotlib 2.2.2
```